src/cnab.rs

```diff
@@ -20,39 +20,47 @@ pub struct Bundle {
     ///
     /// 'install', 'upgrade', and 'uninstall' are default actions, but additional actions
     /// may be defined here.
-    pub actions: Option<HashMap<String, Action>>,
+    pub actions: Option<BTreeMap<String, Action>>,
     /// The list of configurable credentials.
     ///
     /// Credentials are injected into the bundle's invocation image at startup time.
-    pub credentials: Option<HashMap<String, Credential>>,
+    pub credentials: Option<BTreeMap<String, Credential>>,
     /// This field allows for additional data to described in the bundle.
     ///
     /// This data should be stored in key/value pairs, where the value is undefined by
     /// the specification (but must be representable as JSON).
-    pub custom: Option<HashMap<String, serde_json::Value>>,
+    pub custom: Option<BTreeMap<String, serde_json::Value>>,
+
+    /// The JSON Schemata describing the parameters
+    ///
+    /// TODO: Should use a suitable Rust library as the target for this.
```

Comment:
Was the intention to fix this TODO in this PR or later. If later, probably worth associating an issue with the TODO.

Reply:
https://github.com/deislabs/libcnab-rust/issues/25